sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -484,6 +485,23 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("create view as insert into table") {
```

Comment:
Move it to `SparkSqlParserSuite`? 

If we do it here, it will be tested twice.